MariiaNovitska/HW7_functions/task3.py

```diff
@@ -0,0 +1,9 @@
+input_string = input("Write your string: ")
```

Comment:
good job
look at this version of code 

def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result